tests/lib/dbus.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+DISPLAY=:0
+
+# shellcheck source=tests/lib/pkgdb.sh
+. "$TESTSLIB/pkgdb.sh"
+
+init_dbus_env(){
+    export $(cat dbus.env | xargs)
+}
+
+start_dbus_loop_unit(){
```

Comment:
Sure, but that sounds like an implementation detail not worth calling out here. It's usual for things that "start" to "loop", otherwise the function would execute and quit. But we don't say "start browser loop", or "start http server loop", etc.